pandas/_libs/tslibs/util.pxd

```diff
@@ -24,10 +24,10 @@ cdef extern from "Python.h":
     bint PyComplex_Check(object obj) nogil
     bint PyObject_TypeCheck(object obj, PyTypeObject* type) nogil
 
+from numpy cimport int64_t
```

Comment:
I'll trust you if you say this is necessary, but this doesn't make sense to me.  I took the `ctypedef signed long long int64_t` from the numpy `__init__.pxd`, so shouldn't these be equivalent?

(Using the extern version instead of the cimport version is necessary to avoid the NPY_NO_DEPRECATED warnings)

Reply:
I _believe_ "numpy/arrayobject.h" only defines `npy_int64` - it is only the cython pxd that also typedefs this to `int64_t`.

So then this typedef is basically a noop (?)
```c
    ctypedef signed long long int64_t
```

The function failing to compile is this one from util.pxd.  <stdint.h> is normally included in cython generated code, so on all platforms but MSVC 2008 it is providing the necessary int64_t typedef
```
cdef inline int64_t get_nat():
    return NPY_MIN_INT64
```

Reply:
Thanks for clearing this up.  Is there a viable fix that doesn't involve the `from numpy cimport`?

I'm going to merge this here so we can move the master-PR along; hopefully we can get rid of the cimport in the next round of cython-cleaning.